src/rewriters/Rewriter.h

```diff
@@ -31,31 +31,28 @@ class Rewriter {
             unsigned int nextChild = 0;
         };
         // MB: Relies on an invariant that id of a child is lower than id of a parent.
```

Comment:
And the same comment is here, so this should also be removed.

Reply:
It turns out this comment is still relevant (something to discuss, perhaps), but I move it to the description of the method instead. Feels like a better place for it now.